src/llmcompressor/utils/helpers.py

```diff
@@ -1138,3 +1140,18 @@ def preserve_attr(base: object, attr: str):
         yield
     finally:
         setattr(base, attr, value)
+
+
+@contextlib.contextmanager
+def align_modules(modules: Iterable[torch.nn.Module]):
```

Comment:
Why not keep this in compressed tensors with the other cpu offloading tools?

Reply:
Yep! Implementing here before the next CT release

Reply:
https://github.com/neuralmagic/compressed-tensors/pull/282